flink-python/pyflink/java_gateway.py

```diff
@@ -44,6 +43,8 @@ def get_gateway():
                 gateway_port = int(os.environ['PYFLINK_GATEWAY_PORT'])
                 gateway_param = GatewayParameters(port=gateway_port, auto_convert=True)
                 _gateway = JavaGateway(gateway_parameters=gateway_param)
+                # import the flink view
+                import_flink_view(_gateway)
```

Comment:
What about change it as follows:
if 'PYFLINK_GATEWAY_PORT' in os.environ:
                gateway_port = int(os.environ['PYFLINK_GATEWAY_PORT'])
                gateway_param = GatewayParameters(port=gateway_port, auto_convert=True)
                _gateway = JavaGateway(gateway_parameters=gateway_param)
else:
               _gateway = launch_gateway()

import_flink_view(_gateway)

Reply:
Yes.It looks better